tss-esapi/src/context/tpm_commands/non_volatile_storage.rs

```diff
@@ -157,7 +158,35 @@ impl Context {
         }
     }
 
-    // Missing function: NV_Increment
+    /// Increment monotonic counter index
+    ///
+    /// # Details
+    /// This method is used to increment monotonic counter
+    /// in the TPM.
```

Comment:
We're also a bit inconsistent with documentation, but the methods in this file are so similar that it might be worth adding a doc at the module level instead, to show how to use some of the methods.
(Again, not needed for this PR, just a mental note)

Reply:
I have added an issue for the missing code examples in the NV context methods. #359